### ISSUE TITLE
Return confidence/probability percentage from classifyTokens

### ISSUE THREAD

**Issue by tptee** [ayanonagon/Parsimmon#10]
Hi,

Would be awesome if we could access a confidence level/percentage from ParsimmonNaiveBayesClassifer.classifyTokens method, maybe some easily-digestible human-friendly value.

**Comment by ayanonagon** [comment]
I agree. Thanks for the feedback! Always open for pull-requests as well. :octocat:

**Comment by ayanonagon** [comment]
@tptee Not sure if Swift is something that interests you, but I’m in the middle of making a Swift version of Parsimmon, and because Swift supports tuples, the API can be something like:
```swift
let (category, confidence) = classifier.classify("I swear that this is not spam")
```
which would be pretty slick I think. :smile:

**Comment by tptee** [comment]
+1 looks awesome!

I don't have the NLP background to help with algorithms, but I'd love to contribute in some way to your Swift rewrite!
